.github/workflows/ods.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Integration-test
         run: |
           docker-compose -f docker-compose.yml -f docker-compose.it.yml build adapter-it
-          docker-compose -f docker-compose.yml -f docker-compose.it.yml up -d adapter
+          docker-compose -f docker-compose.yml -f docker-compose.it.yml up -d adapter rabbitmq
```

Comment:
Can we add it as docker dependency for the integration tests, so rabbitmq starts up automatically?

Reply:
Good point, that's better!